openassessment/fileupload/backends/swift.py

```diff
@@ -32,10 +35,11 @@ def get_upload_url(self, key, content_type):
         key, url = get_settings()
         try:
             temp_url = swiftclient.utils.generate_temp_url(
-                path='%s/%s/%s' % (url.path, bucket_name, key_name),
+                path='/v%s%s/%s/%s' % (SWIFT_BACKEND_VERSION, url.path, bucket_name, key_name),
```

Comment:
Will this cause any incompatibilities for people who are already using the Swift backend?